cognite/client/_version.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "2.40.1"
-__api_subversion__ = "V20220125"
+__version__ = "2.41.0"
+__api_subversion__ = "V20220216"
```

Comment:
not sure about this subversion thing, do you need newer on for some of services?
should be safe for ark services anyway as I think we don't use date version headers yet

Reply:
I mean it is set per sdk, not per resource :thinking: but anyway, if it explodes tests should be added to prevent it in the future :grin:

Reply:
We don't need it in geospatial. But it felt like the right thing to do since it is in the version file. As you say, if it blows up somewhere, then we'll add a test.

Reply:
@vincent-cognite the __version__ shouldn't have been regressed 😅

Reply:
also this api_subversion is used for beta release afaik so it could've been left unchanged

Reply:
@dmivankov ^^

Reply:
ok, will. set it back.
Any place that explains how this works?

Reply:
hmm, I have no idea either...